### DISCUSSION TITLE
how to interrupt a running action

### DISCUSSION THREAD

**Discussion by bolla99** [limbonaut/limboai#143] [General]
Hi, as far as i know a beaviour tree should allow to interrupt a running action, for example if i have a selector with two child, a condition and an action that becomes running, the condition should be checked each tick and eventually interrupt the consequent running action.
I read the documentation and i found out that it is the intended behavior for the selector to resume the running action without checking the condition again; therefore I ask if and how interruption of a running action can be done.

**Comment by limbonaut** [comment]
We provide that type of selector/sequence too. What you describe is called `DynamicSelector`/`DynamicSequence` in LimboAI. Check [BTDynamicSelector in documentation](https://limboai.readthedocs.io/en/latest/classes/class_btdynamicselector.html).

**Comment by bolla99** [comment]
Thank you! That's exactly what I meant.
